tools/msvs/find_python.cmd

```diff
@@ -0,0 +1,51 @@
+@IF NOT DEFINED DEBUG_HELPER @ECHO OFF
+SETLOCAL
+:: If python.exe is in %Path%, just validate
+FOR /F "delims=" %%a IN ('where python 2^> NUL') DO (
+  SET need_path=0
+  SET p=%%~dpa
+  IF NOT ERRORLEVEL 1 GOTO :validate
+)
+
+:: Query the 3 locations mentioned in PEP 514 for a python2 InstallPath
+FOR %%K IN ( "HKCU\Software", "HKLM\SOFTWARE", "HKLM\Software\Wow6432Node") DO (
+  SET need_path=1
+  CALL :find-main-branch %%K
+  :: If validate returns 0 just jump to the end
+  IF NOT ERRORLEVEL 1 GOTO :validate
+)
+EXIT /B 1
+
+:: Helper subroutine to handle quotes in %1
+:find-main-branch
+SET main_key="%~1\Python\PythonCore"
+REG QUERY %main_key% /s | findstr "2." | findstr InstallPath > NUL
```

Comment:
This prints error messages to my console before finding the correct key... Maybe add `2>&1`?